pipelines/ligtheweight-component-and-container-operations/ligtheweight-component-and-container-operations.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
Is it correct if I understand it as you don't currently use the component that you have created?